lightning/src/ln/peer_handler.rs

```diff
@@ -1981,7 +1981,6 @@ impl<Descriptor: SocketDescriptor, CM: Deref, RM: Deref, OM: Deref, L: Deref, CM
 	// broadcast_node_announcement panics) of the maximum-length addresses would fit in a 64KB
 	// message...
 	const HALF_MESSAGE_IS_ADDRS: u32 = ::core::u16::MAX as u32 / (NetAddress::MAX_LEN as u32 + 1) / 2;
-	#[deny(const_err)]
```

Comment:
No, it's has been warn-by-default for a while now, so specifying this is superfluous. Plus, it recently was even made a hard error, so this will deny-by-default going forward

Reply:
Well, it starts emitting annoying warnings? But alright, will drop the commit then for now. 🤷‍♂️

Reply:
Alright, dropped the commit for now, may reintroduce once warnings showing up in stable.